src/Internal/ProductAttributesLookup/LookupDataStore.php

```diff
@@ -95,7 +95,7 @@ function ( $settings, $section_id ) {
 						$settings[] = array(
 							'title'         => __( 'Enable table usage', 'woocommerce' ),
 							'desc'          => __( 'Use the product attributes lookup table for catalog filtering.', 'woocommerce' ),
-							'id'            => 'woocommerce_attribute_lookup__enable',
+							'id'            => 'woocommerce_attribute_lookup__enabled',
```

Comment:
@Konamiman all options for this feature uses a naming style that isn't used in any other place in the platform, could be nice to make it consistent with all other options.